integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -1,11 +1,19 @@
 const path = require("path")
+process.env["MEDUSA_FF_SALES_CHANNELS"] = true
+
```

Comment:
issue: Shouldn't this be passed by through the `startServerWithEnvironment`? 

Here we are setting it for the entire testing context, which could lead to issues I imagine

Reply:
Yes, ideally. 
The column decorators are loaded and the corresponding metadata is set when the entities are imported. The server works fine without it, but the factories break since the column data is not set correctly (because the environment is set on runtime later).

Do you know if we can re-import entities etc.?

Reply:
Ahh alright.. that is tricky; one thing is for sure though - we can't set the env here as that would modify the context for all tests following this one since we are not resetting the env. 

Will try to dig around a bit...